vocabulary/modified.csv

```diff
@@ -0,0 +1,10 @@
+field_id,name,mapping_approach,date_field_id,modified_by_field_id,modified_by_value,target_concept_id,target_value_concept_id,target_unit_concept_id
+3894,Age heart attack diagnosed,Modified,53,6150,1,40770406,4329847,9448
+3627,Age angina diagnosed,Modified,53,6150,2,40770406,77670,9448
+4056,Age stroke diagnosed,Modified,53,6150,3,40770406,4310996,9448
+2966,Age high blood pressure diagnosed,Modified,53,6150,4,40770406,4086587,9448
```

Comment:
Note that these are specifically _modified numeric_, where the value of the `field_id` goes into the `value_as_number` field. 

This is different from field _20551_, where the value maps to a `value_as_concept_id`.

So we need to separate out these mapping approaches.

Reply:
The reason I put them together is that part of the filed name is used to fill the value_as_concept_id as well in the example. "age heart attack diagnosed" --> value_as_concept_id = 4329847 (myocardial infarction).